docs/docs/05-developer-guide/08-wfspec-development/06-exception-handling.md

```diff
@@ -27,7 +27,7 @@ The syntax to handle a `Failure` is similar no matter which type of `Node` you a
 This section is concerned with throwing an `EXCEPTION` at the `ThreadSpec` level inside a `WfSpec`. If you want to throw an `EXCEPTION` at the Task Worker level, please refer to the [Task Worker Development Docs](/docs/developer-guide/task-worker-development#throwing-workflow-exceptions)
 :::
 
-In programming languages such as Java and Python (please, let's not mention Go...I have opinions about Go but it's best I don't discuss it), you can `throw` or `raise` an `Exception`. For example:
+In well-designed programming languages such as Java and Python, you can `throw` or `raise` an `Exception`. For example:
```

Comment:
I think the 'well-designed' is something we could exclude from these instructions. If we have people that really like GoLang can interpret this in a way we don't want. WDYT?

Reply:
:joy: